### DISCUSSION TITLE
[Question] How to almost maximize a window in the top center?

### DISCUSSION THREAD

**Discussion by bigplayer-ai** [rxhanson/RectanglePro-Community#637] [Ideas]
[Question] How to almost maximize a window in the top center?
Hi,
I want to almost maximize a window in the top center without docking it to the top left corner. 
How can I do this? My problem is with docking... This is the best setting I have found so far, but the docking position is not right.
<img width="931" alt="Screenshot 2023-12-28 at 21 10 20" src="https://github.com/rxhanson/RectanglePro-Community/assets/119108387/b71741a4-ede4-40bf-a31b-993bd3075ba6">

**Comment by rxhanson** [comment]
This is an edge case that isn't accounted for in Custom Size & Position, unfortunately. When I've thought through this before, I didn't have an optimal solution. Initially I was thinking of adding in a configuration for offsets that could do this, but maybe one more Docking Position for "Upper Center" would make more sense - at least for your scenario. I assume that you want there to be some gap above the window, which is why you don't want a "Top" Docking Position.

**Comment by bigplayer-ai** [comment]
> This is an edge case that isn't accounted for in Custom Size & Position, unfortunately. When I've thought through this before, I didn't have an optimal solution. Initially I was thinking of adding in a configuration for offsets that could do this, but maybe one more Docking Position for "Upper Center" would make more sense - at least for your scenario. I assume that you want there to be some gap above the window, which is why you don't want a "Top" Docking Position.

Hey,
any updates about this?

**Reply by rxhanson** [reply]
Is your desired outcome another docking position of Upper Center?

**Reply by bigplayer-ai** [reply]
> Is your desired outcome another docking position of Upper Center?

Yes, I can set up a custom position with Upper Center, or I can add almost maximize offset via terminal.
